contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Why your growth team isn't winning (and how to fix it)
+date: 2023-09-14
+author: ["lior-neu-ner"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/athlete-hog.jpeg
+featuredImageType: full
+category: Product growth
+tags: 
+  - Guides
+  - Product metrics
+  - Product engineers
+---
+
+There are tell-tale signs when a growth team isn't succeeding: Stuck metrics, missed KPIs, and burnt-out engineers. Everyone is frustrated, but how do you fix it?
+
+[Alexey Komissarouk](https://alexeymk.com/), former head of growth engineering at [MasterClass](https://www.masterclass.com/), has seen it all. He grew Masterclass xyz... 
```

Comment:
Alexey are there any achievements or numbers I can add here that can help build up your credibility and authority on the content?

Reply:
Sure - I think alexeymk.com overall is a pretty good place to link, especially for now.

MasterClass - I grew the Growth Eng org from 8 to ~25, and the valuation quadrupled during my 3 years there. That feel about right?

Reply:
let's incldue here that these days Alexey is a [Growth Engineering Advisor](alexeymk.com/growth-eng)

Reply:
> MasterClass - I grew the Growth Eng org from 8 to ~25, and the valuation quadrupled during my 3 years there. That feel about right?

@AlexeyMK  Hmm any other metrics/achievements we could include? I think stating the valuation growth is not an ideal, since we cannot attribute it to you directly.

Reply:
heh, those are the public ones.  MasterClass is pretty tight-lipped about internal ones.

>   we cannot attribute it to you directly.

not inviting you to my next performance review, lol

Reply:
Will rework to focus on your experience in growth. I want to avoid valuations, as 2021 had crazy valuations and people may be cynical